.github/workflows/ci.yml

```diff
@@ -553,6 +554,22 @@ jobs:
             echo "::set-output name=have_azure_secrets::false"
           fi
         id: check-secrets
+      - name: Check Delta Databricks secrets
+        run: |
+          if [[ "${{ secrets.DATABRICKS_73_JDBC_URL }}" != "" && \
+                "${{ secrets.DATABRICKS_91_JDBC_URL }}" != "" && \
+                "${{ secrets.DATABRICKS_TOKEN }}" != "" && \
+                "${{ secrets.DATABRICKS_AWS_SECRET_ACCESS_KEY }}" != "" && \
+                "${{ secrets.DATABRICKS_AWS_ACCESS_KEY_ID }}" != ""
```

Comment:
Let's check only for DATABRICKS_TOKEN secret existence (not AND of all the conditions)
This way it will be harder for the job to accidentally _not run_ when eg we add/remove new versions we test against, or make a typo in one of the secrets' names